data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/RuntimeUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.scalar.db.dataloader.core.util;
+
+import static com.scalar.db.common.error.CoreError.DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT;
+
+/** Utils for runtime checks */
+public class RuntimeUtil {
+
+  /**
+   * Argument null check
+   *
+   * @param values List of arguments
+   * @throws NullPointerException when one of the arguments is null
+   */
+  public static void checkNotNull(Object... values) {
+    for (Object value : values) {
+      if (value == null) {
+        throw new NullPointerException(DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT.getMessage());
```

Comment:
```suggestion
    for (Object value : values) {
      if (value == null) {
        throw new NullPointerException(DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT.buildMessage());
```

Reply:
I have updated the change as suggested.
Thank you.